test/system/experimental.test.ts

```diff
@@ -121,15 +123,15 @@ test("static scan for Identifier type image (nginx:1.19.0)", async (t) => {
       "nginx" in pluginResult.scannedProjects[0].depTree.dependencies,
     "Contains some expected dependency",
   );
-  t.deepEqual(
-    pluginResult.plugin.imageLayers,
-    [
-      "29a4ff5c2250ab72c60545ba67bd67d87daa05e4abd186e7d488d921287c893b/layer.tar",
-      "bd0f0d2dba229cb9005123b54a646fe612882343c06b68c4722f221fdc597d82/layer.tar",
-      "5a88377adcb6f00f0388d45a929fe1159916d7c9e733db78d6defb3fcd325a68/layer.tar",
-      "94f68a38cdeff495ef63de6533b7d7486312ff7bc184120a620ad738536f548d/layer.tar",
-      "44fc3f4af0d0f3587192b742d77b7c1c85cfd230bfb5474f6e1619da7962e3e3/layer.tar",
-    ],
-    "Layers are read correctly",
+  t.same(
+    pluginResult.plugin.imageLayers.length,
+    5,
+    "Returns expected number of layers",
+  );
+  t.ok(
+    pluginResult.plugin.imageLayers.every((layer) =>
+      layer.endsWith("layer.tar"),
+    ),
+    "Every found layer has the correct name",
   );
```

Comment:
Since this test will run only on Windows and not on Linux, we probably don't need to do this - we can compare layers like before

Reply:
yeah, 100%!!